### ISSUE TITLE
predicted instance disappears after doubleclick

### ISSUE THREAD

**Issue by smidm** [murthylab/sleap#414]
sleap version:			1.0.9

steps to reproduce:

1. start SLEAP
2. add a video
3. save the project to a folder with `models`
4. predict -> run inference
5. wait until inference finishes
6. go -> next labeled frame
7. double click on an instance

result: the instance disappears with the following traceback
```
Traceback (most recent call last):
  File ".../sleap/gui/app.py", line 1090, in _after_plot_update
    overlay.add_to_scene(self.state["video"], frame_idx)
  File ".../sleap/gui/overlays/instance.py", line 46, in add_to_scene
    self.player.addInstance(instance=instance)
  File ".../sleap/gui/widgets/video.py", line 437, in addInstance
    instance = QtInstance(instance=instance, player=self, **kwargs)
  File ".../sleap/gui/widgets/video.py", line 1768, in __init__
    self.updateBox()
  File ".../sleap/gui/widgets/video.py", line 1842, in updateBox
    rect = self.getPointsBoundingRect()
  File ".../sleap/gui/widgets/video.py", line 1822, in getPointsBoundingRect
    min((point.x for point in points)), min((point.y for point in points))
ValueError: min() arg is an empty sequence
```

**Comment by talmo** [comment]
Hi @smidm,

Would you happen to have the `.slp` file saved after doing inference?

Thanks!

**Comment by smidm** [comment]
https://ufile.io/u9q6r986

**Comment by talmo** [comment]
Hi @smidm,

I wasn't able to reproduce this error when creating instances from the predictions in that .slp file.

Do you know at what frame this happens at?

Also, what are the dimensions of the video frames in this dataset?

**Comment by smidm** [comment]
I found out that the bug is not reproducible after loading the project with the predicted instances. Probably the structures are rebuild correctly after loading. I packed all the data for reproducing the bug here: http://cmp.felk.cvut.cz/~smidm/bug/bug_predicted_instance_disappears.zip

1. load slp (just project with a video)
2. predict -> run inference -> current frame
3. double click on an instance
4. the instance disappears with the traceback above

**Comment by talmo** [comment]
Hi @smidm,

Thanks for the test data!

It looks like the problem happens when doing inference on projects without a skeleton. The skeleton gets merged appropriately in the labels (which is why it works when you save and re-open), but it doesn't update the GUI so you get an error when creating instances from predictions.

I've implemented a fix for this in alpha version [`v1.0.10a9`](https://github.com/murthylab/sleap/releases/tag/v1.0.10a9) if you want to try it out. Otherwise you should be able to get it working by either saving and re-opening the project before trying to create instances from predictions.
